### ISSUE TITLE
Unable to re-size D-Lite height 

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#3719]
Currently it is not possible to change the height of D-Lite by dragging-up or dragging-down like this 
[![Image from Gyazo](https://i.gyazo.com/cecb2abcf9a12e4e3b35df654fd56722.gif)](https://gyazo.com/cecb2abcf9a12e4e3b35df654fd56722)
this is when I try to drag-up and down

[![Image from Gyazo](https://i.gyazo.com/7a245c806b7e28bd4562296aa5c3fea8.gif)](https://gyazo.com/7a245c806b7e28bd4562296aa5c3fea8)
This is when I try to resize both width and height - as can be seen only the width is changing.

**Comment by IanMayo** [comment]
Yes, ideally we would override the current window resize behaviour with this:
* before the window resizes, retrieve the current viewport (in data coords)
* as the window resizes, always ensure the viewport remains the same (over-write with saved coords)

**Comment by helenayele** [comment]
Currently I can drag and increase/decrease both width and height of D-Lite, but the map /track inside the map doesn't change its size when D-Lite changes its dimensions.
[![Image from Gyazo](https://i.gyazo.com/b14ae2e44713cf58bb7dce278098f814.gif)](https://gyazo.com/b14ae2e44713cf58bb7dce278098f814)

**Comment by IanMayo** [comment]
This new GeoTools feature solves this issue:
https://github.com/geotools/geotools/pull/2324

Once it's merged into GeoTools, we can switch to that version of GeoTools, and us the new layout option.

**Comment by helenayele** [comment]
Currently I can re-size  D-Lite and change its height by dragging down and up but to some level. 
is this by design?
[![Image from Gyazo](https://i.gyazo.com/ca2b85ffd9f2cf9f15d67d15e1d14b79.gif)](https://gyazo.com/ca2b85ffd9f2cf9f15d67d15e1d14b79)

**Comment by ayeshare** [comment]
@helenayele @IanMayo 
![image](https://user-images.githubusercontent.com/38917949/83726203-cfb46a00-a660-11ea-9174-289c72d1723d.png)

See image on right side. thats the smallest I could get to.  But there is an initial size with the map and we cant go lesser than that.  That might be intentional.  
I also remember making some changes to make sure that map doesn't get hidden under the ribbon tabs

**Comment by IanMayo** [comment]
In the prioritised ToDo, I've put this beneath `refactor D-Lite`.  Once D-Lite is refactored, and the map-related code is encapsulated, it will be easier to find 3rd party support for fixing this issue.

**Comment by helenayele** [comment]
Okay I will come back to this.
